src/Client/StoreOnChainWallet.php

```diff
@@ -214,6 +214,34 @@ public function getStoreOnChainWalletTransaction(
         }
     }
 
+    public function updateStoreOnChainWalletTransaction(
+        string $storeId,
+        string $cryptoCode,
+        string $transactionId,
+        ?string $comment
+    ): StoreOnChainWalletTransaction
+    {
```

Comment:
cs fixer complains about this curly brace not beeing on the line above (like on other methods) https://github.com/btcpayserver/btcpayserver-greenfield-php/actions/runs/4194164802/jobs/7330438070

Reply:
so if you see this and can fix, would be great otherwise I will merge tomorrow and fix it in an upcoming pr